src/nidmm/metadata/functions.py

```diff
@@ -1509,7 +1518,7 @@
         'method_templates': [
             {
                 'documentation_filename': 'lock',
-                'library_interpreter_filename': 'default_method',
```

Comment:
I don't think this is a new enough export. The init method starting on line 1377 should have the new initialization template stuff right?

Reply:
You're right, I need to update metadata. However, now that I do that...
`TypeError: Only simple types allowed for arrays: ViAddr`
🙁

Reply:
Done. But using non-exported metadata at the moment. I'll open a separate conversation for that.